MagickCore/compress.h

```diff
@@ -31,6 +31,7 @@ typedef enum
   DXT1Compression,
   DXT3Compression,
   DXT5Compression,
+  BC7Compression,
```

Comment:
I haven't had time to do a proper review of this PR yet will try to do that this weekend. Already a quick remark about this change. New values can only be added at the end.